pkg/reaper/datacenter_test.go

```diff
@@ -28,13 +212,11 @@ func TestAddReaperSettingsToDcConfig(t *testing.T) {
 			&cassandra.DatacenterConfig{
 				Meta:    api.EmbeddedObjectMeta{Name: "dc1"},
 				Cluster: "cluster1",
-				Users:   []cassdcapi.CassandraUser{{SecretName: "cluster1-reaper", Superuser: true}},
 				PodTemplateSpec: &corev1.PodTemplateSpec{
 					Spec: corev1.PodSpec{
 						InitContainers: []corev1.Container{{
-							Name:            "jmx-credentials",
-							Image:           "docker.io/busybox:1.33.1",
-							ImagePullPolicy: corev1.PullIfNotPresent,
+							Name:  "jmx-credentials",
+							Image: "docker.io/busybox:1.33.1",
```

Comment:
We'll need to make this image configurable in order to support using different or private registries.

Reply:
I added a field to control this image, but I realize that this field cannot be honored when using standalone Reaper resources. This is because the field contents are only read and applied by k8ssandra-controller, not by reaper-controller. Is that OK for you?

Reply:
FYI this field has moved to `CassandraClusterTemplate`. 

https://github.com/k8ssandra/k8ssandra-operator/blob/f221f13604589355cabe14c50e2d375aedc81470/apis/k8ssandra/v1alpha1/k8ssandracluster_types.go#L181-L186